pypesto/__init__.py

```diff
@@ -20,40 +22,16 @@
     NegLogPriors,
     ObjectiveBase)
 from .problem import Problem
-from .petab import (
-    PetabImporter)
-from . import startpoint
 from .result import (
     Result,
     OptimizeResult,
     ProfileResult,
     SampleResult)
-from .optimize import (
-    minimize,
-    OptimizeOptions,
-    OptimizerResult,
-    Optimizer,
-    ScipyOptimizer,
-    IpoptOptimizer,
-    DlibOptimizer,
-    PyswarmOptimizer)
-from .profile import (
-    parameter_profile,
-    ProfileOptions,
-    ProfilerResult)
-from .sampling import (
-    sample,
-    Sampler,
-    InternalSampler,
-    MetropolisSampler,
-    AdaptiveMetropolisSampler,
-    ParallelTemperingSampler,
-    AdaptiveParallelTemperingSampler,
-    Pymc3Sampler,
-    McmcPtResult,
-    geweke_test)
-from .engine import (
-    SingleCoreEngine,
-    MultiThreadEngine,
-    MultiProcessEngine)
+
+# import simple modules as submodules
+from . import engine
+from . import logging
+from . import petab
```

Comment:
Do we want to import that by default? Comes with heavy libsbml.

Reply:
valid point. let's not.